libs/domain/product/src/services/adapter/normalizers/product-list/product-list.normalizer.ts

```diff
@@ -59,6 +60,11 @@ export function productFacetNormalizer(
     1
   );
 
+  const ratingFacet = data[0].rangeFacets!.splice(
+    data[0].valueFacets!.findIndex((v) => v.name === 'rating'),
+    1
+  );
```

Comment:
why we can't just use `find` method?

Reply:
We need also remove the rating component from original array